FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -419,7 +419,8 @@ def getInstallations( self, argss ):
           key = 'UnInstallationTime.smaller'
         elif arg == '-uta':
           key = 'UnInstallationTime.bigger'
-      else:
+      elif isinstance(key, str):
```

Comment:
What are you doing here ? if key is an str you cast it to str ?
Btw, basestring maybe ?